### DISCUSSION TITLE
Is it possilbe to disable the pop up(transition)  animation ?

### DISCUSSION THREAD

**Discussion by q2333gh** [File-New-Project/EarTrumpet#1163] [Q&A] [answered]
I wan t no transition animations so it will be more clear and resource saving.Simple ,fast.
I konw some how its in the source code .But im still a fish of coder.Didnt know how to find it.

**Comment by themrvideouser** [comment]
Have you tried disabling Animations in Windows Settings in general? 
I don't remember ever having seen a pop up from EarTrumpet.

**Comment by riverar** [comment, marked as answer]
As @themrvideouser mentioned, we do respect the animation settings in Windows. That said, the flyout seems to still be animating for me so looks like there's a bug here...

![image](https://user-images.githubusercontent.com/475132/182631209-e0e1596b-aa2c-4a92-bc98-1b6c6b00cbdf.png)
